samples/msal-browser-samples/VanillaJSTestApp2.0/app/wamBroker/README.md

```diff
@@ -0,0 +1,9 @@
+# WAM Broker Sample
+
+This sample can be used to acquire hardware bound tokens from WAM using the [Windows 10 Accounts Chrome extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji).
```

Comment:
```suggestion
This sample can be used to acquire device bound tokens from WAM using the [Windows 10 Accounts Chrome extension](https://chrome.google.com/webstore/detail/windows-10-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji).
```

Reply:
Also please expand WAM and mention this can be used only for Windows at this point